README.md

```diff
@@ -1 +1,68 @@
-# space-travelers-hub
\ No newline at end of file
+# Space Travelers' Hub
+
+>  Space Travelers' Hub is to a web application for a company that provides commercial and scientific space travel services. This application allows users to book rockets and join selected space missions.
+
+## Features
+- Uses real live data from the SpaceX API.
+- Book Rockets.
+- Join selected space missions.
+
+## Built Using
+
+- ReactJs.
+- Redux
+- React Router
+- CSS.
+- GitFlow.
+
+## Run the app in the development mode(Port 3000)
+
+```
+npm start
+```
+
+## Launches the test runner
+
+```
+npm test
+```
+
+## Builds the app for production
+
+```
+npm run build
+```
+
+## Demo App
+
+**live demo** [To be Published.]()
```

Comment:
- [ ] Please make sure to deploy your website and put the link in the README file.

Reply:
Hello @ShahierNashaat! Thanks for your review. I cannot seem to get a hold of my partner who told me he would do the deployment of the site (since it is his repo we are working on) and I am worried as you can imagine.
I kindly have to ask if the deployment of the site is NOT OPTIONAL since I cannot find it in the project requirements :[React & Redux group project - Space Travelers' Hub](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md). All other changes requested above have been made. Note the linters are passing, only the Netlify deployment linters are not. He is not responding to my calls to delete it from his Netlify or try another site to deploy it into and I am finding myself in a difficult situation for my finishing this week.
Thanks for your understanding!

Reply:
You can try [heroku](https://www.heroku.com/)

Reply:
Can I deploy form someone else`s repository? As you can see in the image below from Heroku I cannot choose any other repo except my own or microverseenic and this project is in my partners repo, not mine. 
![image](https://user-images.githubusercontent.com/88782702/175784052-13bf7145-b96b-4a67-a760-36bf6ef34570.png)

Reply:
I think no sorry, actually I don't know what to say but you can try to delete the netlify linters.

Reply:
And also I think that the code reviewer won't matter to him the netlify linters and if the project deployed correctly